distpi/README.md

```diff
@@ -0,0 +1,78 @@
+Distributed Pi
+================================================================================
+
+This directory contains an implementation of the [Bailey-Borwein-Plouffe formula]
```

Comment:
[Bailey-Borwein-Plouffe formula] or [[Bailey-Borwein-Plouffe]
not linked

Reply:
doh. fixed.